### PR TITLE
Fixed Optional Chaining chapter's issues

### PR DESCRIPTION
**Pull request by vandadnp**

### CODE REVIEW

**Comment by vandadnp** [comment]
@dakeshi could you review this please whenever you have the time?

This addresses the comments that you left here: https://github.com/crowdsourcebooks/theswiftbook/commit/bcf4580c7d9b02184673b4a06383181e9ad2921d

🙇

**Comment by dakeshi** [comment]
I'd be glad to. 😄

File: chapters/optional_chaining.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ func processNameFor(person: Person?){
 }
 ```
 
-This can easily get quite cumbersome to type and difficult understand. The solution to this is using Swift's optional chaining capabilities to read the `value` field of the `name` property of the `Person` instance only if the person exist and has a valid `Name` instance set for it.
+This can easily get quite cumbersome to type and difficult understand. The solution to this is using Swift's optional chaining capabilities to the optional values safely.
```
Comment (dakeshi) [inline chapters/optional_chaining.md]: It's easy to understand. 👍

File: chapters/optional_chaining.md [hunk 2]
```diff
@@ -66,6 +66,8 @@ The general syntax for optional chaining is:
 let someOptionalValue = optional1?.optional2?.optional3
 ```
 
+The return value of optional chaining is always an optional value. You can then [safely unwrap](optionals.md#safely-unwrapping) the optional to read its value.
+
```
Comment (dakeshi) [inline chapters/optional_chaining.md]: Good. 👍

**Comment by vandadnp** [comment]
@dakeshi are you now happy with this PR? Should I proceed to merge it? 😎🤗

**Comment by dakeshi** [comment]
@vandadnp. Great job. Thanks for your PR. 👏

**Comment by vandadnp** [comment]
Thanks @dakeshi for going through it and giving me valuable feedback. I'm merging now 🙌 😄

**Pull request merged**